justspaces/urls.py

```diff
@@ -22,4 +22,11 @@
     url(r'^admin/', admin.site.urls),
     url(r'^pong/$', pong,),
     url(r'', include('justspacesapp.urls')),
+    # DB Store plugin URLs
+    url(r'^fobi/plugins/form-handlers/db-store/',
+        include('fobi.contrib.plugins.form_handlers.db_store.urls')),
+    # View URLs
+    url(r'^fobi/', include('fobi.urls.view')),
+    # Edit URLs
+    url(r'^fobi/', include('fobi.urls.edit')),
```

Comment:
Can we customize the template for editing a form? I do not think we need to show handlers.

<img width="606" alt="screen shot 2019-02-26 at 1 37 51 pm" src="https://user-images.githubusercontent.com/13078679/53437628-fb5f9380-39cb-11e9-89a7-48f978941ffa.png">

Reply:
Agreed! I was going to leave this until I figure out how to set a default handler, because right now you have to add them manually.

Reply:
okay - got it.